docs/man_pages/project/creation/create.md

```diff
@@ -16,4 +16,12 @@ Creates a new project for native development with NativeScript from the default
 * `<App Name>` is the name of project. The specified name must meet the requirements of all platforms that you want to target. <% if(isConsole) { %>For more information about the `<App Name>` requirements, run `$ tns help create`<% } %><% if(isHtml) { %>For projects that target Android, you can use uppercase or lowercase letters, numbers, and underscores. The name must start with a letter.  
 For projects that target iOS, you can use uppercase or lowercase letters, numbers, and hyphens.<% } %>    
 * `<App ID>` is the application identifier for your project. It must be a domain name in reverse and must meet the requirements of all platforms that you want to target. If not specified, the application identifier is set to `org.nativescript.<App name>` <% if(isConsole) { %>For more information about the `<App ID>` requirements, run `$ tns help create`<% } %><% if(isHtml) { %>For projects that target Android, you can use uppercase or lowercase letters, numbers, and underscores in the strings of the reversed domain name, separated by a dot. Strings must be separated by a dot and must start with a letter. For example: `com.nativescript.My_Andro1d_App`  
-For projects that target iOS, you can use uppercase or lowercase letters, numbers, and hyphens in the strings of the reversed domain name. Strings must be separated by a dot. For example: `com.nativescript.My-i0s-App`.<% } %>
\ No newline at end of file
+For projects that target iOS, you can use uppercase or lowercase letters, numbers, and hyphens in the strings of the reversed domain name. Strings must be separated by a dot. For example: `com.nativescript.My-i0s-App`.<% } %>
+
+<% if(isHtml) { %> 
+### Related Commands
+
+Command | Description
+----------|----------
+[init](init.html) | Initializes a project for development. It will ask you interactively to configure the project and will write a package.json for you.
```

Comment:
install command is in `project/configuration` section and the init command is in `project/creation`. Do I need to relate them?